src/Illuminate/View/Compilers/Concerns/CompilesConditionals.php

```diff
@@ -119,4 +119,25 @@ protected function compileEndAuth()
     {
         return '<?php endif; ?>';
     }
+
+    /**
+     * Compile the if-errors statements into valid PHP.
+     *
+     * @param  string|null  $key
+     * @return string
+     */
+    protected function compileErrors($key = null)
+    {
+        return "<?php if(errors({$key})): ?>";
```

Comment:
This is still using the old ``` errors ``` helper, also we can't pass ``` $bag ``` I think you could change ``` $key ``` to accept an array too. This way:

```

@errors
Has errors
@enderrors

@errors('name')
Name field has errors
@enderrors

@errors(['bag' => 'login'])
Login form has errors
@enderrors

@errors(['bag' => 'login', 'key' => 'name'])
Name field of login form has errors
@enderrors